src/controllers/sold.land.filter.controller.ts

```diff
@@ -16,8 +17,12 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
 
     const appData: ApplicationData = getApplicationData(req.session);
 
+    let backLinkUrl: string = config.LANDING_PAGE_URL;
```

Comment:
Can we reuse the `getSoldLandFilterBackLink` from navigation util, it will simplify the logic in the controller

Reply:
Good point, I'll do that now